pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -171,3 +175,21 @@ func (m *defaultVirtualCenterManager) IsExtendVolumeSupported(ctx context.Contex
 	}
 	return !is67u3Release, nil
 }
+
+// IsOnlineExtendVolumeSupported checks if online extend volume is supported or not.
+func (m *defaultVirtualCenterManager) IsOnlineExtendVolumeSupported(ctx context.Context, host string) (bool, error) {
+	log := logger.GetLogger(ctx)
+
+	// Get VC instance
+	vcenter, err := m.GetVirtualCenter(ctx, host)
+	if err != nil {
+		log.Errorf("Failed to get vCenter. Err: %v", err)
+		return false, err
+	}
+	vCenterVersion := vcenter.Client.Version
```

Comment:
Does the vCenter version gets updated when the vCenter server is upgraded? I think we need QE team to test this scenario where the CSI dynamically starts supporting online volume resize once the vCenter server is upgraded to 7.0 U2. Can you follow up with QE team to add a scenario for this?

Reply:
Created an internal bug for FVT to track this